bundles/AdminBundle/Resources/public/js/pimcore/startup.js

```diff
@@ -166,7 +166,7 @@ Ext.onReady(function () {
 
         var jsonData = null;
         try {
-            jsonData = Ext.decode(response.responseText);
+            jsonData = Ext.decode(response.responseJson);
```

Comment:
@weisswurstkanone What if the response isn't JSON? Maybe check for both properties here? 
Probably related to this here, right: https://forum.sencha.com/forum/showthread.php?471298-Undocumented-breaking-change-responseText-undefined ?